exampleapp/App.jsx

```diff
@@ -20,6 +22,29 @@ var App = React.createClass({
   changeLeafRadius(evt)  { this.setState({ leafRadius: parseInt(evt.target.value) });  },
   changeLeafSpacing(evt) { this.setState({ leafSpacing: parseInt(evt.target.value) }); },
 
+  categoryCheckboxUpdated(checkbox, checked) {
+    var filter = this.state.filter;
+    filter[checkbox] = !checked;
+    this.setState({ filter });
+  },
+
+  generateCheckboxes() {
+    return Object.keys(categories).map(cat => {
+      var category = categories[cat];
+      var checked = this.state.filter[category] || false;
+      return (
+        <fieldset>
```

Comment:
I'm seeing some React warnings about blind maps, so I guess this is missing a `key={cat}`